crates/c-api/src/linker.rs

```diff
@@ -120,3 +120,31 @@ pub unsafe extern "C" fn wasmtime_linker_get_default(
         *func = Box::into_raw(Box::new(HostRef::new(linker.store(), f).into()))
     })
 }
+
+#[no_mangle]
+pub extern "C" fn wasmtime_linker_get_one_by_name(
+    linker: &mut wasmtime_linker_t,
```

Comment:
I think this should be `&wasmtime_linker_t` as well, right?

Reply:
As yes, thanks. Updated along with the `wasmtime_linker_get_default()` API as well.